atlas-web/src/main/java/uk/ac/ebi/gxa/web/ui/plot/ExperimentPlot.java

```
@@ -139,7 +135,11 @@ public class ExperimentPlot {
             }
         }
 
-        prepareBoxAndWhiskerData(ewd.getExpressionStatistics(ad, deIndices));
+        try {
+            prepareBoxAndWhiskerData(ewd.getExpressionStatistics(ad, deIndices));
+        } catch (StatisticsNotFoundException e) { 
+            // ignore
```

Hm.. not sure its good idea to ignore the exception here